.github/workflows/upload_release_asset.yml

```diff
@@ -13,6 +13,7 @@ jobs:
           FOSSA_API_KEY: ${{secrets.FOSSAAPIKEY}}
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    permissions: {}
```

Comment:
@ceddy4395 We need to specify the permissions we want here right? What happens if this object is empty? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idpermissions

Reply:
Ah yes I see where it gets uploaded, I've added `contents: write` to the permission which will allow the uploading.